src/ctapipe/reco/impact.py

```diff
@@ -85,51 +90,59 @@ class ImPACTReconstructor(Component):
 
     """
 
+    minimiser = traits.CaselessStrEnum(
+        ["minuit", "l-BFGS"],
+        default_value="minuit",
+        help="name minimiser to use in the fit",
+    ).tag(config=True)
+
+    use_time_gradient = traits.Bool(
+        default_value=False, help="Use time gradient in ImPACT reconstruction"
+    ).tag(config=True)
+
+    root_dir = traits.Unicode(
```

Comment:
Better name `template_directory`?

Reply:
Yes. Maybe even removing the hard coded names for the templates is worth a look

Reply:
That could be a `TelescopeParameter(Path)`